### PR TITLE
some update

### PR DESCRIPTION
**Pull request by fydne**
- Some optimize
- Changed: **!(is) => is not**
- == null => is null
- != null => is not null
- etc..

### CODE REVIEW

File: Synapse/Api/CustomObjects/SynapseRagdollObject.cs [hunk 1]
```diff
@@ -19,22 +19,24 @@ public SynapseRagdollObject(RoleType role, DamageType damage, Vector3 pos, Quate
         }
         internal SynapseRagdollObject(SynapseSchematic.RagdollConfiguration configuration)
         {
-            Ragdoll = CreateRagDoll(configuration.RoleType, configuration.DamageType, configuration.Position, Quaternion.Euler(configuration.Rotation), configuration.Scale, configuration.Nick);
+            Ragdoll = CreateRagDoll(configuration.RoleType, configuration.DamageType, configuration.Position,
+                Quaternion.Euler(configuration.Rotation), configuration.Scale, configuration.Nick);
```
Comment (AlmightyLks) [inline Synapse/Api/CustomObjects/SynapseRagdollObject.cs:23]: Our screens are wider than 500px, if anything, I'd appreciate a
```cs
Ragdoll = CreateRagDoll(configuration.RoleType, 
			configuration.DamageType, 
			configuration.Position,
			Quaternion.Euler(configuration.Rotation), 
			configuration.Scale, 
			configuration.Nick);
```

File: Synapse/Api/Enum/DoorType.cs [hunk 2]
```diff
@@ -44,13 +42,6 @@ public enum DoorType
...
-        [Obsolete("012 is no longer in the game", true)]
-        LCZ_012 = 0,
-        [Obsolete("012 is no longer in the game", true)]
-        LCZ_012_Bottom = 1,
-        [Obsolete("012 is no longer in the game", true)]
-        LCZ_012_Locker = 2,
+        Other
     }
-}
+}
```
Comment (AlmightyLks) [inline Synapse/Api/Enum/DoorType.cs:47]: Causes a breaking change, don't remove

File: Synapse/Api/Enum/DoorType.cs [hunk 3]
```diff
@@ -44,13 +42,6 @@ public enum DoorType
...
         PrisonDoor,
         Airlock,
-        Other,
-
-        [Obsolete("012 is no longer in the game", true)]
-        LCZ_012 = 0,
-        [Obsolete("012 is no longer in the game", true)]
-        LCZ_012_Bottom = 1,
-        [Obsolete("012 is no longer in the game", true)]
-        LCZ_012_Locker = 2,
```
Comment (AlmightyLks) [inline Synapse/Api/Enum/DoorType.cs:54]: Causes a breaking change, don't remove

File: Synapse/Api/Enum/Effect.cs [hunk 4]
```diff
@@ -55,10 +51,6 @@ public enum Effect
         /// </summary>
         /// <remarks>0 = Disabled, 1 = Enabled</remarks>
         Disabled,
-        [Obsolete("Disarmed is no longer in the game", true)]
-        Disarmed,
-        [Obsolete("Discharge is no longer in the game", true)]
-        Discharge,
```
Comment (AlmightyLks) [inline Synapse/Api/Enum/Effect.cs:61]: Don't remove.

File: Synapse/Api/Enum/LockerType.cs [hunk 5]
```diff
@@ -2,11 +2,12 @@
 {
     public enum LockerType
     {
+        Default = -1,
```
Comment (AlmightyLks) [inline Synapse/Api/Enum/LockerType.cs:5]: What for?

File: Synapse/Api/Events/PlayerEvents.cs [hunk 6]
```diff
@@ -158,8 +158,14 @@ internal void InvokePlayerLeaveEvent(Player player)
...
-                Allow = allow, Player = userSetup.GetPlayer(), IntercomTalk = intercom, RadioTalk = radio,
-                Scp939Talk = scp939, ScpChat = scpChat, SpectatorChat = specChat, DissonanceUserSetup = userSetup
+                Allow = allow,
+                Player = userSetup.GetPlayer(),
+                IntercomTalk = intercom,
+                RadioTalk = radio,
+                Scp939Talk = scp939,
+                ScpChat = scpChat,
+                SpectatorChat = specChat,
+                DissonanceUserSetup = userSetup
```
Comment (AlmightyLks) [inline Synapse/Api/Events/PlayerEvents.cs:168]: Touché, no idea how I overlooked that

File: Synapse/Api/Enum/LockerType.cs (hunk 5 above)
Comment (fydne) [reply Synapse/Api/Enum/LockerType.cs:5]: https://github.com/SynapseSL/Synapse/pull/109/files#diff-3b00d15d5022fdc700a7c6dfb4c142403729c66c42ee54df438a1405f22b8a37R65

Comment (fydne) [reply Synapse/Api/Enum/LockerType.cs:5]: Room.cs 65 line

Comment (fydne) [reply Synapse/Api/Enum/LockerType.cs:5]: Locker*

**Comment by fydne** [comment]
okay

File: Synapse/Api/Locker.cs [hunk 7]
```diff
@@ -55,22 +55,28 @@ public Vector3 Scale
...
+                LockerType _get()
+                {
+                    if (Name.Contains("AdrenalineMedkit")) return LockerType.MedkitWallCabinet;
+                    else if (Name.Contains("RegularMedkit")) return LockerType.AdrenalineWallCabinet;
+                    else if (Name.Contains("Pedestal")) return LockerType.ScpPedestal;
+                    else if (Name.Contains("MiscLocker")) return LockerType.StandardLocker;
+                    else if (Name.Contains("RifleRack")) return LockerType.RifleRackLocker;
+                    else if (Name.Contains("LargeGunLocker")) return LockerType.LargeGunLocker;
+                    return LockerType.Default;
+                };
```
Comment (AlmightyLks) [inline Synapse/Api/Locker.cs:76]: @GrafDimenzio 
Question, can the `Locker#Name` ever change once spawned?

Comment (GrafDimenzio) [reply Synapse/Api/Locker.cs:76]: No the name is just fetched from the gameobject and can't (normally) be changed

Comment (AlmightyLks) [reply Synapse/Api/Locker.cs:76]: Well then why the hell do you re-evaluate it every time at runtime 😄

Comment (GrafDimenzio) [reply Synapse/Api/Locker.cs:76]: I think Warkis did implement this in a PR and the others accepted the change in version 2.8.0 so I never actually inspected the code for the locker object

Comment (AlmightyLks) [reply Synapse/Api/Locker.cs:76]: Welp

**Comment by AlmightyLks** [comment]
I have some closing words.
The general idea of refactoring some code conventions and styling is appreciated and honestly overdue, however we cannot consider this PR, as half of its contents breaks the current dev environment and causes a breaking change for consumers

In addition to that, this PR barely has any substance to be considered, especially with lazy progress being committed as "some update" and "added some [Obselete]".

Though we will add some refactorings and especially the hiccups you've come to notice into our backlog 😄   

Keep an eye out on anything else that can be PR'ed in the future though! But please keep it somewhat clean. 😄

**Pull request closed without merging**